tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/ThreadSamplingTests.cs

```diff
@@ -0,0 +1,38 @@
+// Modified by Splunk Inc.
+
+#if !NETFRAMEWORK
+
+using Datadog.Trace.TestHelpers;
+using FluentAssertions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests
+{
+    public class ThreadSamplingTests : TestHelper
+    {
+        public ThreadSamplingTests(ITestOutputHelper output)
+            : base("ThreadSampling", output)
+        {
+            SetServiceVersion("1.0.0");
+        }
+
+        [SkippableFact]
+        [Trait("Category", "EndToEnd")]
+        [Trait("RunOnWindows", "True")]
+        public void SubmitTheadSamples()
+        {
+            SetEnvironmentVariable($"SIGNALFX_THREAD_SAMPLING_ENABLED", "true");
+            SetEnvironmentVariable($"SIGNALFX_THREAD_SAMPLING_PERIOD", "1000");
```

Comment:
```suggestion
            SetEnvironmentVariable($"SIGNALFX_THREAD_SAMPLING_PERIOD", "100");
```

so that the `Thread.Sleep` in the sample app can be decreased to `0.5 sec` 

ps. the same change would need to be done in `tracer/test/test-applications/integrations/Samples.ThreadSampling/Properties/launchSettings.json`

Reply:
The real period is the max (SIGNALFX_THREAD_SAMPLING_PERIOD, 1000). We cannot do this :)